### PR TITLE
WNP 98 for DE/FR (Fixes #11211)

### LINKED ISSUE #11211: WNP 98 - EU
## Description

Design: https://www.mozilla.org/en-US/firefox/96.0/whatsnew/

Locales:  DE & FR 

Copy: https://docs.google.com/document/d/13b-AqtjiD-JrtpNrO4ZuINNr935SUaieAtxD8McPER8/edit#heading=h.9ejy2rvy4tnp

Lets use the QR code variation (that was the winner in the last WNP).

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds a WNP for Firefox 98 in German and French.

## Issue / Bugzilla link
#11211

## Testing
- http://localhost:8000/de/firefox/98.0/whatsnew/
- http://localhost:8000/fr/firefox/98.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx98-mobile-de.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx98-mobile-de.html:19]: lovely to see these in use on new code already 🎉

File: media/css/firefox/whatsnew/whatsnew-98-mobile-eu.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.c-main-content {
+    .mzp-c-emphasis-box {
+        margin: 0 auto;
+    }
+
+    .send-to-device-input {
```
Comment (maureenlholland) [inline media/css/firefox/whatsnew/whatsnew-98-mobile-eu.scss:20]: suggestion: remove unused rule (qr codes only on these pages, no input)

**Review by maureenlholland — CHANGES_REQUESTED** [review]
One minor cleanup request, otherwise this all looks great!

**Comment by alexgibson** [comment]
@maureenlholland thanks for the review, updated

**Review by maureenlholland — APPROVED** [review]
r+ 👍

**Pull request merged**